src/android/src/main/java/io/callstack/react/fbads/AdSettingsManager.java

```diff
@@ -25,8 +31,12 @@ public String getName() {
     }
 
     @ReactMethod
-    public void addTestDevice(String deviceHash) {
-        AdSettings.addTestDevice(deviceHash);
+    public void addTestDevice() {
+        SharedPreferences sp = context.getSharedPreferences("FBAdPrefs", 0);
```

Comment:
Why not just `getReactApplicationContext().getSharedPreferences("FBAdPrefs", 0);` instead of saving context in variable?